assets/js/base/components/store-notices-container/index.js

```diff
@@ -25,43 +26,37 @@ const getWooClassName = ( { status = 'default' } ) => {
 	return '';
 };
 
-const StoreNoticesContainer = ( { className, notices, componentId } ) => {
+const StoreNoticesContainer = ( { className, notices } ) => {
 	const { removeNotice } = useStoreNoticesContext();
 	const wrapperClass = classnames( className, 'wc-block-components-notices' );
+	const instanceId = useInstanceId( StoreNoticesContainer );
 
 	return (
-		<div className={ wrapperClass } key={ componentId }>
-			{ notices.map(
-				(
-					props = { id: '', content: '', isDismissible: false },
-					i
-				) => (
-					<Notice
-						key={ 'store-notice-' + i }
-						{ ...props }
-						className={ classnames(
-							'wc-block-components-notices__notice',
-							getWooClassName( props )
-						) }
-						onRemove={ () => {
-							if ( props.id && props.isDismissible ) {
-								removeNotice( props.id );
-							}
-						} }
-					>
-						{ props.content }
-					</Notice>
-				)
-			) }
+		<div className={ wrapperClass } key={ instanceId }>
```

Comment:
You don't need to have a key on the container (I should have caught this earlier).  Keys are only needed on children within a specific component. My first review I was thinking the instanceId was being used for an actual ID, but it looks like it's not needed at all here?